core/package.json

```diff
@@ -28,6 +24,6 @@
     "url": "git@github.com:juspay/presto-ui.git"
   },
   "author": "JUSPAY Technologies <info@juspay.in>",
-  "license": "AGPL",
+  "license": "AGPL-3.0",
```

Comment:
This is AGPL.

Reply:
There is no valid license name as AGPL.
We use AGPL-3.0.
https://github.com/juspay/presto-ui/blob/master/LICENSE#L24

Identifier: https://spdx.org/licenses/AGPL-3.0.html